pynsee/utils/_get_credentials.py

```diff
@@ -53,13 +55,16 @@ def _get_credentials():
 @lru_cache(maxsize=None)
 def _warning_credentials(string):
     if string == "envir_var_used":
-        print(
-            "!!! Existing environment variables used, instead of locally saved credentials !!!"
+        logger.debug(

```

Comment:
should be warning

Reply:
Why should this be ? If the user set environmental variables with those specific keys, this should be intentional ? (Maybe info though...)

Reply:
in the case of pynsee, it should be something the person did, but we don't know when (it might be something from a long time ago that they forgot about) so I'd still advise warning

cc @hadrilec